md/themes.md

```diff
@@ -28,7 +28,28 @@ Any CSS, images or JavaScript files can be shared using a theme resource.
 
 Export the examples to see how to structure a theme.
 
-The default themes are based on the [bootswatch example](https://bootswatch.com/), so you can easily choose another existing theme and package it to define a new theme. 
+### Using bootswatch for a custom theme
+
+The default themes are based on the [bootswatch example](https://bootswatch.com/), so you can easily choose another existing theme and package it to define a new theme.
```

Comment:
Not all of them are based on bootswatch
bootswatch example**s**

Reply:
The "based on bootswatch" was there before, I just moved it, should I change it to say that some default themes?

Reply:
Yes "Some of the themes available are based"

Reply:
done :)